edn/src/symbols.rs

```diff
@@ -25,8 +25,8 @@ pub struct PlainSymbol(pub String);
 pub struct NamespacedSymbol {
     // We derive PartialOrd, which implements a lexicographic based
     // on the order of members, so put namespace first.
-    pub namespace: String,
-    pub name: String,
+    _namespace: String,
```

Comment:
I think this is going to be address in later commits, but it's not common in Rust code to prepend `_` for privacy, since Rust has a compiler with rich visibility support.